### ISSUE TITLE
GLB/GLTF model will not display correctly in Google AR but shows no errors in other viewers

### ISSUE THREAD

**Issue by lampmaker** [google/model-viewer#1651]
I'm using three.js to generate a 3D model and export it as GLB or GLTF using THREE.GLTFExporter. Goal is to make it viewable on our site using google AR.
[https://kaboomlaser.com/pages/ar-test](https://kaboomlaser.com/pages/ar-test) (link to files)

The model looks perfectly fine when I view it in https://sandbox.babylonjs.com/ or other GLTF/glb viewers.

See here: [https://gltf-viewer.donmccurdy.com/#model=https://cdn.shopify.com/s/files/1/0290/5459/9273/files/test.glb](https://gltf-viewer.donmccurdy.com/#model=https://cdn.shopify.com/s/files/1/0290/5459/9273/files/test.glb)

but when I open it in Googles AR scene previewer [https://vr.google.com/scene-viewer-preview](https://vr.google.com/scene-viewer-preview) , it ends up empty. Other models I tried did show up in scene previewer, but with the meshes all spiky and weird.

You can try it with a different model on our site: [(https://kaboomlaser.com/products/test](https://kaboomlaser.com/products/test)
Opening the model in the regular viewer works, but in AR mode it doesn't display correctly.

Below the code I used to reproduce the issue with a simple model. (This outputs the models in the link above).
I did try changing the dimensions of the model, but that didn't help.

```
var x = 0, y = 0;
    var heartShape = new THREE.Shape();
    heartShape.moveTo(x + 5, y + 5);
    heartShape.bezierCurveTo(x + 5, y + 5, x + 4, y, x, y);
    heartShape.bezierCurveTo(x - 6, y, x - 6, y + 7, x - 6, y + 7);
    heartShape.bezierCurveTo(x - 6, y + 11, x - 3, y + 15.4, x + 5, y + 19);
    heartShape.bezierCurveTo(x + 12, y + 15.4, x + 16, y + 11, x + 16, y + 7);
    heartShape.bezierCurveTo(x + 16, y + 7, x + 16, y, x + 10, y);
    heartShape.bezierCurveTo(x + 7, y, x + 5, y + 5, x + 5, y + 5);

    var geometry = new THREE.ExtrudeBufferGeometry(heartShape, {
        depth: 4, bevelEnabled: false,
        bevelThickness: 0.4,
        bevelSize: 0.4,
        steps: 2,
        BevelSegments: 2,
        curveSegments: 10
    });

    var material = new THREE.MeshLambertMaterial({ color: 0xa00A0A });
    var mesh = new THREE.Mesh(geometry, material);

    var scene = new THREE.Scene();
    scene.add(mesh);

    const options = {
        binary: true,
        forceIndices: true
    };

    var exporter = new GLTFExporter();

    exporter.parse(scene, function (data) {
        if (options.binary) {
            savetoFile(data, 'test.glb', 'model/gltf-binary');
        }
        else {
            savetoFile(JSON.stringify(data), 'test.gltf', 'text/plain');
        }
    },
    options);
```

### Browser Affected

- [x ] Chrome
- [ x] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ x] Windows

**Comment by elalish** [comment]
@tpsiaki, this look like a scene-viewer specific issue.

**Comment by elalish** [comment]
@lampmaker Would you mind telling me why you prefer Scene-viewer to WebXR for AR? I'm trying to determine what we need to add to close the gap with native.

**Comment by lampmaker** [comment]
@elalish I just barely understand what I'm doing, but I'll explain the process:
- we have a webshop that can display 3D objects and use AR to view them in a room.    I created the code that allowed met to export our designs into GLB, uploaded them into our webshop and noticed that it will display fine until you view it in AR mode.

https://kaboomlaser.com/products/test  :  clicking on the model works well, but the AR view doesn't work. (tested on android phone)

so I tried to reproduce the issue and found that https://vr.google.com/scene-viewer-preview gave me the same issues.

And that's where I'm at.  Our webshop is hosted by Shopify.       And that is pretty much the extent of my knowledge about this..   Not a real programmer here.  :-)
